core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -27,13 +30,20 @@
         implements Renderer<String>
 {
     private static final JsonCodec<JsonRenderedNode> CODEC = JsonCodec.jsonCodec(JsonRenderedNode.class);
+    private static final JsonCodec<Map<PlanFragmentId, JsonPlanFragment>> PLAN_MAP_CODEC
+            = JsonCodec.mapJsonCodec(PlanFragmentId.class, JsonPlanFragment.class);
```

Comment:
@arhimondr  regarding your comment about using a list instead of map, I thought of using json map format for both EXPLAIN and EXPLAIN ANALYZE (working on its implementation)

Reply:
Could you please explain why does it have to be map for explain analyze?

Reply:
I thought a map would be better if we want to query the json output for a particular key. In explain analyze, we would have keys such as `outputLayout` (which represents the output columns for that stage) and `stageStats` which will have the cpu and mem info. Then, consumers of that output would just require to iterate over the json and query particular keys. 

I didn't really think more about list vs map.

Reply:
That makes sense. Thanks for the explanation.

Reply:
@arhimondr  thank you for the reviews.